### PR TITLE
(SIMP-3923) Created profile to enhance libkv::consul

### PR DESCRIPTION
**Pull request by nick-markowski**
SIMP-3923 #comment simp_consul profile created
SIMP-3924 #close
SIMP-3919 #close
SIMP-3926 #close
SIMP-3949 #close

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,25 @@
+simp_consul - 
+
+Per Section 105 of the Copyright Act of 1976, these works are not entitled to
```
Comment (trevor-vaughan) [inline LICENSE]: This section can go. This is all non-government and copyright held by Onyx Point, Inc.

File: files/consul-acl [hunk 2]
```diff
@@ -0,0 +1,133 @@
+#!/bin/sh
```
Comment (trevor-vaughan) [inline files/consul-acl:1]: Why `sh` instead of `ruby`?

**Review by trevor-vaughan — DISMISSED** [review]

File: files/consul-acl (hunk 2 above)
Comment (nick-markowski) [reply files/consul-acl:1]: Didn't write it or re-evaluate it, not sure.

File: LICENSE (hunk 1 above)
Comment (nick-markowski) [reply LICENSE]: Ok

**Comment by nick-markowski** [comment]
I still need to add client docs, but everything else is ready for review

File: files/consul-acl (hunk 2 above)
Comment (heliocentric) [reply files/consul-acl:1]: It was originally a much simpler shell script

**Comment by nick-markowski** [comment]
TODO: move in consul_members fact from libkv

**Review by heliocentric — APPROVED** [review]

**Pull request merged**